src/ec/packages/ec-component-social-media-share/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ecl-twig/ec-component-social-media-share",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Social Media Share",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
```

Comment:
please add a devdependency to the vanilla component as well (see checklist in PR description)